cmd/utils/flags.go

```diff
@@ -998,6 +998,12 @@ var (
 		Value:  chaindatafetcher.DefaultMaxProcessingDataSize,
 		EnvVar: "KLAYTN_CHAINDATAFETCHER_MAX_PROCESSING_DATA_SIZE",
 	}
+	ChainDataFetcherRangeFetchingTracerFlag = cli.StringFlag{
+		Name:   "chaindatafetcher.rangefetching.tracer",
+		Usage:  "Chaindatafetcher range fetching tracer (callTracer or fastCallTracer)",
```

Comment:
What about saying clearly that only two tracers are available when range fetching chaindatafetcher?
```suggestion
		Usage:  "Chaindatafetcher range fetching tracer (only callTracer or fastCallTracer is available)",
```